.github/workflows/ci.yml

```diff
@@ -351,6 +350,39 @@ jobs:
         run:  |
           cd build
           timeout --signal=SIGABRT 60m ./tst/webrtc_client_test
+
+  mbedtls-ubuntu-gcc-static:
```

Comment:
This just verifies that it builds, but I don't see any checks (e.g. objdump) to verify that the binary is _actually_ static or whether it is able to run the binary.

Reply:
The build fails. That is the verification. With mbedtls, if the library types are mixed on ubuntu, the build is not successful.